index.js

```diff
@@ -3,6 +3,7 @@ import VideosApi from './videos';
 
 const {ZiggeoPlayer} = NativeModules;
 const {ZiggeoRecorder} = NativeModules;
+const {ZVideoViewModule} = NativeModules;
```

Comment:
We have a separate class, which encapsulates native video view (player) functionality and allows to use it as a widget in code: `video_view.js`. Events should be alse passed to this widget

Reply:
okay